jobs-shell/tests/e2e-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -eux pipefail
+
+# Remove me
+PROJECT_ID=$(gcloud config get project)
+
+gcloud config set run/region "${_REGION}"
+
+# Generate a random 15 character alphanumeric string (lowercase only)
+RAND_ID=$(LC_CTYPE=C tr -dc 'a-z0-9' < /dev/urandom | head -c 15)
```

Comment:
This use to be a restriction on name length. We may be able to use BUILD_ID or a cloud build substitution instead of generating our own.

Reply:
I did like the short identifiers. BUILD_ID is _long_